src/fleet/azext_fleet/_params.py

```diff
@@ -29,20 +29,31 @@ def load_arguments(self, _):
         c.argument('enable_vnet_integration', action='store_true', is_preview=True, help='Whether to enable apiserver vnet integration for the Fleet hub or not.')
         c.argument('apiserver_subnet_id', validator=validate_apiserver_subnet_id, is_preview=True, help='The subnet to be used when apiserver vnet integration is enabled. It is required when creating a new Fleet with BYO vnet.')
         c.argument('agent_subnet_id', validator=validate_agent_subnet_id, is_preview=True, help='The ID of the subnet which the Fleet hub node will join on startup. If this is not specified, a vnet and subnet will be generated and used.')
-        c.argument('enable_managed_identity', action='store_true', help='Enable system assigned managed identity (MSI) on the Fleet resource.')
-        c.argument('assign_identity', validator=validate_assign_identity, help='With --enable-managed-identity, enable user assigned managed identity (MSI) on the Fleet resource. Specify the existing user assigned identity resource.')
+        c.argument('mi_system_assigned', action='store_true', help='Enable system assigned managed identity (MSI) on the Fleet resource.')
+        c.argument('mi_user_assigned', validator=validate_identity_id, help='The resource ID of an existing user assigned identity.')
```

Comment:
Does this help text make sense?

Reply:
E.g. don't we want something like 'Enable user managed identity ... with this resource ID'

Reply:
Improved.